doc/tutorials/using_zephyr_as_uos.rst

```diff
@@ -3,12 +3,13 @@
 Run Zephyr as the User VM
 #########################
 
-This tutorial describes how to run Zephyr as the User VM on the ACRN hypervisor. We are using
+This tutorial describes how to run Zephyr as the User VM on the ACRN
+hypervisor. We are using
 Kaby Lake-based Intel NUC (model NUC7i5DNHE) in this tutorial.
 Other :ref:`ACRN supported platforms <hardware>` should work as well.
 
 .. note::
-   This tutorial uses the (default) SDC scenario. If you use a different
+   This tutorial uses the default SDC scenario. If you use a different
```

Comment:
This tutorial mentions a Kaby Lake NUC that isn't listed in the Supported Hardware doc and SDC scenario. What's the plan for updating the tutorial in a future release?

Reply:
That NUC was in the supported HW doc when this tutorial was written (v2.2).  The "default SDC" scenario is a bigger problem since that doesn't exist any more for that board.  Oddly, we don't mention the SDC scenario later in this doc when building ACRN?  Let's open an issue for this tutorial as needing an update given the development changes that have happened since v2.2.  I suspect we should abandon this PR for now until we can update it correctly. @gvancuts  @NanlinXie

Reply:
I opened an issue: https://github.com/projectacrn/acrn-hypervisor/issues/6831, and I will close this PR.